homeassistant/components/websocket_api/decorators.py

```diff
@@ -31,7 +31,9 @@ def async_response(
     @wraps(func)
     def schedule_handler(hass, connection, msg):
         """Schedule the handler."""
-        hass.async_create_task(_handle_async_response(func, hass, connection, msg))
+        # As the webserver is now started before the start
+        # event we do not want to block for websocket responders
```

Comment:
This is going to be a problem for anything spawned by an API interaction by the user while Home Assistant is starting up.

Reply:
I'm thinking long running logbook api calls or history api calls could block startup for a few seconds.  Is there anything else that is a problem hot spot that I'm not thinking about?

Reply:
Two things I can think of:
 - Triggering a script that takes X time
 - Starting a config flow for new integration that starts installation of a big requirements (like `zwave`)

Reply:
I don't think that it would be a big problem, eventually all tasks should stop and no websocket API should start long running tasks